cmd/nerdctl/container/container_logs_test.go

```diff
@@ -95,6 +95,9 @@ func TestLogsWithInheritedFlags(t *testing.T) {
 	base.Cmd("run", "-d", "--name", containerName, testutil.CommonImage,
 		"sh", "-euxc", "echo foo; echo bar").AssertOK()
 
+	// It appears this test flakes out with Docker seeing only "foo\n"
+	// Tentatively adding a pause in case this is just slow
+	time.Sleep(time.Second)
```

Comment:
Seems a bug in Docker?

Reply:
Maybe - if that does not help, I ll just mark it skipped for docker